extension.js

```diff
@@ -30,12 +28,16 @@ setInterval(updateCounter, pollInterval * (60 * 1000));
 init();
 
 function openCurrentURL() {
+    function hasAssignedPullRequests() {
```

Comment:
since i cant make a private function i put it into this function. hope you are ok with that?

Reply:
doesn't matter.